modules/develop/partials/run-transforms.adoc

```diff
@@ -508,6 +577,35 @@ fn filter_valid_json(event: WriteEvent, writer: &mut RecordWriter) -> Result<()>
 	}
 	Ok(())
 }
+```
+--
+JavaScript::
++
+--
+```js
+import { onRecordWritten, Record } from "@redpanda-data/transform-sdk";
+
+onRecordWritten(filterValidJson);
+
+function filterValidJson(event, writer) {
+  const recordValue = event.record.value;
+
+  if (isValidJson(recordValue)) {
+    writer.write(event.record);
+  } else {
+    writer.write(event.record, ??toTopic("invalid-json"));
```

Comment:
@rockwotj or @oleiman I couldn't find a `toTopic()` method in the SDK. Have we added this yet?